src/Pretzel.Logic/Configuration.cs

```diff
@@ -56,16 +60,16 @@ private void EnsureDefaults()
                 _config.Add("permalink", DefaultPermalink);
             }
 
-            _defaultsConfiguration = new DefaultsConfiguration(_config);
+            Defaults = new DefaultsConfiguration(_config);
         }
 
-        [OnImportsSatisfied]
-        internal void ReadFromFile()
+        public void ReadFromFile(string path)
```

Comment:
To avoid multiple reads, what do you think of making this method static and creating a `Configuration`?

Reply:
I think it is doable if `CommandCollection` exports it's configuration.

Reply:
Maybe I am missing something, but why don't just create it in program and add it to MEF?
Or is it not permitted in MEF2?

Reply:
That is only possible if you are inside an `OnImportSatisfied` method. But we don't know which command get's executed, so we have no way of exporting it in `CommandCollection.OnImportsSatisfied`. But there is only a single call to it.
I see another benifit: If the configuration file changes we now can reload it in the `taste` command.

Reply:
And no we cant create it in program. But it's so early in the lifecycle so i think it's fine there.

Reply:
Ok, I'd prefer to do otherwise but since we are restricted by mef2 that will do.
And the benefit is great!